app/javascript/components/admin/AdminTable.js

```diff
@@ -61,7 +61,7 @@ class AdminTable extends React.Component {
   createAuditButton(data) {
     const rowData = data.rowData;
     return (
-      <div id={rowData.id} className="float-left edit-button">
+      <div id={rowData.id} className="float-left edit-button" aria-label="Create Audit Row Button">
```

Comment:
I think there could be a better label for this button since technically it doesn't create any new audits, instead it opens the Audit modal for the associated user. Perhaps "Show Audits Button" or something similar might work?

Reply:
good call, I updated the name